src/responseHandlers/AbstractResponseHandler.js

```diff
@@ -33,6 +36,27 @@ var AbstractResponseHandler = jsface.Class([EventEmitter], {
 	},
 
 	_printResponse: function(error, response, body, request) {
+        if (Globals.outputFileVerbose) {
+            var filepath = path.resolve(Globals.outputFileVerbose);
+
+            var requestString = "-------------------------------------------------------------------------------------------\n" +
```

Comment:
Sure, I wonder where else templates could be applicable in newman. @digital-abyss, any specific places you have in mind where you think using templates would be better?